oci-seccomp-bpf-hook.go

```diff
@@ -183,7 +193,21 @@ func startFloatingProcess() error {
 		return err
 	}
 	pid := s.Pid
-	fileName := s.Annotations["io.containers.trace-syscall"]
+	var outputFile string
+	var inputFile string
+	annotation := s.Annotations["io.containers.trace-syscall"]
+	annotationSplit := strings.SplitN(annotation, ";", 2)
```

Comment:
I just noticed it now. We should do some validation here and make sure that the specified value of the annotation has the required format. I tend to be very picky about parsing (it's very common to regress on that) and I think we should refactor the code into a dedicated function `func parseAnnotation(annotation string) (outputFile string, inputFile string, err error)` and create an `oci-seccomp-bpf-hook_test.go` file where we can create a unit test for this function.

A nice example for such a unit test is here: https://github.com/containers/image/blob/master/pkg/sysregistriesv2/system_registries_v2_test.go#L78

The input may have more than two ";", that should error. If we have two, then both should be absolute paths.